src/components/product/ProductsList.component.tsx

```diff
@@ -1,19 +1,22 @@
 import React, { useEffect, useState } from 'react';
 
+import Pagination from '@/components/product/Pagination.component.tsx';
 import styles from '@/components/product/Product.module.css';
 import ProductCard from '@/components/product/ProductCard.component.tsx';
 import SearchBar from '@/components/search_bar/SearchBar.component.tsx';
 import type Product from '@/interfaces/Products.ts';
 
-interface ProductsListProps {
-    apiUrl: string;
-}
-
-const ProductsList: React.FC<ProductsListProps> = ({ apiUrl }) => {
+export default function ProductsList() {
+    const apiUrl = 'https://ma-backend-api.mocintra.com/api/v1/products?category&name=Shoes';
```

Comment:
Api url must be without query params by default: https://ma-backend-api.mocintra.com/api/v1/products
Also would be better to move it to constants file.